lib/spirv/test/general_shaders/simple.glsl

```diff
@@ -0,0 +1,11 @@
+#version 320 es
```

Comment:
It looks like these files are code and can have comments, so we should put a copyright header at the top of them.

Reply:
we should also make sure the license script knows about .glsl files

Reply:
Added glsl case to filesystem.dart